### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by diegog-sf**

### CODE REVIEW

**Review by josehernandez-sf — APPROVED** [review]
+1 - Fantastic intro!

**Review by leandroforte-sf — APPROVED** [review]
+1 - Great work! Nothing to add from my side.

File: README.md [hunk 1]
```diff
@@ -1,5 +1,19 @@
...
+Welcome to Superformula's Coding challenge, we are excited to see what you can build!
+
+This take home tests aims to evaluate your skills in building a Flutter application. We are looking for a well-structured and well-tested application that demonstrates your knowledge of Flutter and the Dart language.
+
+We are not looking for pixel perfect designs, but we are looking for a well-structured application that demonstrates your skills and best practices developing a flutter aplication. We know there are many ways to solve a problem, and we are interested in seeing how you approach this one. If you have any questions, please don't hesitate to ask.
+
+Things we'll be looking on your submission:
+- App structure for scallability
+- Error and optional (?) handling
+- Widget three optimization
```
Comment (edson-mgm) [inline README.md]: fix: should be widget "tree" optimization

**Review by edson-mgm — CHANGES_REQUESTED** [review]

**Review by mathieu-sf — APPROVED** [review]
+1 I will try it at hime :)

File: README.md [hunk 2]
```diff
@@ -1,5 +1,19 @@
 # Restaurant Tour
 
+Welcome to Superformula's Coding challenge, we are excited to see what you can build!
+
+This take home tests aims to evaluate your skills in building a Flutter application. We are looking for a well-structured and well-tested application that demonstrates your knowledge of Flutter and the Dart language.
```
Comment (dionesxxx) [inline README.md]:
```suggestion
This take home test aims to evaluate your skills in building a Flutter application. We are looking for a well-structured and well-tested application that demonstrates your knowledge of Flutter and the Dart language.
```

**Review by dionesxxx — COMMENTED** [review]
Looks very clear to me Diego.

**Pull request merged**